interface/modules/custom_modules/oe-module-prior-authorizations/public/index.php

```diff
@@ -161,6 +162,29 @@ function refreshme() {
                             print TABLE_TD . text($iter['end_date']);
                         }
                         print TABLE_TD . text($iter['cpt']);
+
+                        $initialUnits = (int)$iter['init_units'];
+
+                        if ($initialUnits > 0) {
+                            $percentRemaining = round(($remaining / $initialUnits) * 100);
+                        } else {
+                            $percentRemaining = 0;
+                        }
+                        $barColor = '#4CAF50'; // Green: Full/Plenty remaining
+                        if ($percentRemaining <= 66) {
+                            $barColor = '#ffc107'; // Yellow: Getting low
+                        }
+                        if ($percentRemaining <= 33) {
+                            $barColor = '#dc3545'; // Red: Very low
+                        }
```

Comment:
Consider using a match expression here. Note that if $percentRemaining <= 33 it's also <= 66, so the assignment is done three times in that case.